include/alpaka/queue/QueueHipRtBlocking.hpp

```diff
@@ -106,7 +106,7 @@ namespace alpaka
                     dev::DevHipRt const m_dev;   //!< The device this queue is bound to.
                     hipStream_t m_HipQueue;
 
-#if BOOST_COMP_HCC  // NOTE: workaround for unwanted nonblocking hip streams for HCC (NVCC streams are blocking)
+#if BOOST_COMP_HCC || BOOST_COMP_HIP  // NOTE: workaround for unwanted nonblocking hip streams for HCC (NVCC streams are blocking)
```

Comment:
In the comment should probably also be `HCC or HIP`?

Reply:
Same for other occurrences of this comment (next files).

Reply:
I will remove this for HIP-clang and will add a `static_assert` that we currently not support callbacks with `HIP-clang`.
This line is only a workaround because the hip callbacks are not synchronous. The current workaround adds polling but this is destroying all performance e.g. for PIConGPU.